### DISCUSSION TITLE
I Can't able to add content to docs.

### DISCUSSION THREAD

**Discussion by chandra1199** [outline/outline#5590] [Self Hosting]
I have installed outline and integrated with zitadel cloud for sso. After I logged in i have created a new doc, in that doc i am able to add title but not able to add content. I have installed 0.70.2 version of outline.

**Comment by bvlbhargav** [comment]
@chandra1199 
if you are nginx please check the configuration of the same with the documentation.
This looks like issue with the configuration

**Reply by chandra1199** [reply]
Below I am providing my outline configuration file. Please check and let me know if i miss any thing.

name: outline
image:
  repository: outlinewiki/outline
  tag: 0.69.1
  pullPolicy: IfNotPresent
replicas: 1
env:
  OIDC_CLIENT_ID: 223262838511173889@grafana
  OIDC_CLIENT_SECRET: bIrylgVoaVNlFDgHfO0b60MqDypepvcIcUP5z6bzDwiztPyXlcgUfY5MiAAk1NBU
  OIDC_AUTH_URI: https://chandra-iizvuw.zitadel.cloud/oauth/v2/authorize
  OIDC_TOKEN_URI: https://chandra-iizvuw.zitadel.cloud/oauth/v2/token
  OIDC_USERINFO_URI: https://chandra-iizvuw.zitadel.cloud/oidc/v1/userinfo
  OIDC_USERNAME_CLAIM: preferred_username
  OIDC_DISPLAY_NAME: Zitadel
  OIDC_SCOPES: "openid offline_access profile email"
ingress:
  enabled: true
  host: outline.192.168.1.21.nip.io
  annotations: 
    kubernetes.io/ingress.class: nginx
  tls:
    enabled: false

redis:
  enabled: true
  architecture: "standalone"
  persistence:
    size: 2Gi
    storageClass: ""
  auth:
    enabled: false

postgresql:
  enabled: true
  architecture: "standalone"
  postgresqlDatabase: "outline"
  postgresqlUsername: "outline"
  postgresqlPassword: "outline"
  postgresqlPostgresPassword: "outline"
  persistence:
    enabled: true
    storageClass: ""
    size: 1Gi

minio:
  enabled: true
  disableWebUI: false
  ingress:
    enabled: true
    hostname: data.outline.192.168.1.21.nip.io
    certManager: false
    tls: false
    annotations:
      kubernetes.io/ingress.class: nginx
  mode: "standalone"
  defaultBuckets: "ol-data"
  secretKey:
    password: "Uk4LuwVFvRDiG6k4nYA5C^835V" 
  accessKey:
    password: "zFPVNNK*QNp3^V4J38Gy76SehG"
  persistence:
    enabled: true
    storageClass: ""
    size: 2Gi

**Comment by chandra1199** [comment]
After i deploy outline, I have created ingress for outline with nginx class. I have installed nginx ingress controller using below command.

helm upgrade --install nginx-ingress ingress-nginx/ingress-nginx   --namespace ingress-nginx
